miniserver/routes/index.js

```diff
@@ -58,25 +58,25 @@ const parseSMS = async (req, res, next) => {
         return respond(`error: pay request requires a valid amount': ${smsParams[2]} example request "p +14158401999 0.1"`)
       }
       const amount = ethers.utils.parseEther(smsParams[2])
-      let toAddress
+      let funderAddress
       // Allow requesting of funds from users by address (without checking registered phone number)
       if (smsParams[1].substr(0, 2) === '0x') {
         if (!isValidAddress(smsParams[1])) {
           return respond(`error: invalid funder address ${smsParams[1]}. example request "p 0x8ba1f109551bd432803012645ac136ddd64dba72 0.1"`)
         }
-        toAddress = checkSumAddress(smsParams[1])
+        funderAddress = checkSumAddress(smsParams[1])
```

Comment:
smsParams[1] is destination address, not "funder" (from)

Reply:
**In the current logic** 
a. the requestor is driven by the phone number
b. the funder is driven by the phone number of address.

**The call to the send function is as follows**
```
   function send(
        uint256 amount,
        address from,
        address to
   ```
**testing uses the twilio cli as follows** 
`'twilio api:core:messages:create --from "+14158401410" --to "+17372327333" --body "p 4158401410 0.1"'`
**So current mapping is**
sms params 0 : drives the transaction type i.e. p maps to send
sms params 1 : is the funder address (i.e. from in the send function)
sending phone number: is the to (i.e. the to in the send function)
sms params 2 : is the amount (i.e amount in the send function)

**We can change this mapping if needed or document this somewhere to make it more clear.**

Reply:
@polymorpher can you confirm that you are happy with the above mapping.
If not can you let me know the changes or documentation that is required 🙏

Reply:
Not sure whether we are on the same page of what "funder" implies? Is it not the person who initiated and funded the transaction? If so, why is smsParams[1] not the phone number of the user whom the initiator is paying to?

Reply:
Sorry for the confusion. Yes the funder is the person who funded the transaction (the user). 
**But the funding request (payment) was initiated by the requestor (the creator).** 
I originally had this confused as well.
This is why we use the above mappings.

Let me try and clarify the above by adding a little more context and using the terms `user` and `creator`
* `user` : someone who registers their phone and makes deposits into the miniwallet and approves those funds to be used by the creator. (also known as funder).  For testing we use 
  * Phone: `+14158401410`
  * Address: `0x143A933E79931006b3Eb89cBc938587546faF159`
* `creator` the game developer/artist who registers their phone and requests the operator to send the users approved funds (also known as the requestor). For Testing we use 
  * Phone: `+16505473175`
  * Address: `0x58bB8c7D2c90dF970fb01a5cD29c4075C41d3FFB`
  * Caller: `TOKEN WARRIOR` (used when constructing contract calls via https://demo.smswallet.xyz)

The complete scenario is as follows
1. The user (funder) and creator(requestor) register their phones.
2. The user (funder) transfers 40 ONE into their account.
3. The user(funder) deposits 10 ONE into the miniwallet and approves the creator to withdraw some of these funds (1 ONE). 
  * This can be triggered by a request url such [as this](https://localhost:3100/call?amount=10&callback=aHR0cHM6Ly9sb2NhbGhvc3Q6MzA5OS9jYWxsYmFjaw%3D%3D&calldata=eyJtZXRob2QiOiJhcHByb3ZlKGFkZHJlc3MsdWludDI1NikiLCJwYXJhbWV0ZXJzIjpbeyJuYW1lIjoic3BlbmRlciIsInR5cGUiOiJhZGRyZXNzIiwidmFsdWUiOiIweDU4YkI4YzdEMmM5MGRGOTcwZmIwMWE1Y0QyOWM0MDc1QzQxZDNGRkIifSx7Im5hbWUiOiJhbW91bnQiLCJ0eXBlIjoidWludDI1NiIsInZhbHVlIjoiMTAwMDAwMDAwMDAwMDAwMDAwMCJ9XX0%3D&caller=Token%20Warrior&comment=Fund%20MiniWallet%2010%20ETH%20and%20approve%201%20ETH%20for%20creator%200x58bB8c7D2c90dF970fb01a5cD29c4075C41d3FFB&dest=0xe7f1725E7734CE288F8367e1Bb143E90bb3F0512). 
  * Which can be created by using the [demo request page](https://demo.smswallet.xyz/request) 
  * This gives the user a request which looks as follows (once again depositing funds into the mini-wallet and approving some of those funds to be used by the creator).
![FundingRequest](https://user-images.githubusercontent.com/1572027/191806183-a1be35a3-c76a-459c-8492-621aca8effa7.png)

4. The user (funder) can then check their **minwallet** balance by text `b` from their phone (+14158401410)
  * We test this using the  twilio command 
  * `twilio api:core:messages:create --from "+14158401410" --to "+17372327333" --body "b"`
  * And receive the following response
  * `Your balance is 10.0 (address: 0x143a933e79931006b3eb89cbc938587546faf159)`

5. The creator (requestor) can than ask the operator to send them some funds on behalf of the user(funder) by sending a message from their phone (+16505473175)
  * We test this using the twilio command
  * `twilio api:core:messages:create --from "+16505473175" --to "+17372327333" --body "p 4158401410 0.1"`
  * And receive the following response
  * `Sent 0.1 (from: 0x143a933e79931006b3eb89cbc938587546faf159, to 0x58bb8c7d2c90df970fb01a5cd29c4075c41d3ffb, transaction:0x83b98c966385060236d7d7c4c7eff0d69fade1912e652c8efd24d533c20450ea)`
 * **The creator (requestor) balance is now increased by 0.1**
<img width="503" alt="CreatorBalance" src="https://user-images.githubusercontent.com/1572027/191808764-2a863289-2d89-41fd-bcab-4ceca17702dc.png">

  * The creator (requestor) still has no funds held in the miniwallet (the funds were sent from the miniwallet to their address)
  * We test the creator checking their **miniwallet** balance using the twilio command (i.e. from the creator (requestor))
  * `twilio api:core:messages:create --from "+16505473175" --to "+17372327333" --body "b"`
  * and receive the response
  * `Your balance is 0.0 (address: 0x58bb8c7d2c90df970fb01a5cd29c4075c41d3ffb)
* **The user (funders) **miniwallet** balance is now decreased**
*   * We test this using the  twilio command 
  * `twilio api:core:messages:create --from "+14158401410" --to "+17372327333" --body "b"`
  * And receive the following response
  * `Your balance is 9.9 (address: 0x143a933e79931006b3eb89cbc938587546faf159)`
  * The user (funder) sms-wallet balance after the funding and approval is as follows
![UserBalance](https://user-images.githubusercontent.com/1572027/191810952-46183b2f-3da9-482f-a4b8-903907ee8865.png)

Reply:
After discussion with @polymorpher 
The requirement is that the user calls the payment function (not the creator).
Will update the codebase accordingly.

Reply:
Have updated the codebase so that user now sends payment requests.
Also updated workflow above and testing documents in FEATURE_ROLLOUT.md